qubes-rpc/qvm-copy-to-vm.kde

```diff
@@ -20,21 +20,35 @@
 #
 #
 
+if type kdialog 2> /dev/null; then
+kdlg=1
+else kdlg=0
+fi
+
+if [ $kdlg -eq 1 ] ; then
 VM=$(kdialog -inputbox "Enter the VM name to send files to:")
+else
+VM=$(zenity --entry --text="Enter the VM name to send files to:")
+fi
+
 if [ X$VM = X ] ; then exit 0 ; fi
 
 SIZE=$(du --apparent-size -c -- "$@" 2> /dev/null | tail -1 | cut -f 1)
+if [ $kdlg -eq 1 ] ;then
 REF=$(kdialog --progressbar "Copy progress")
+fi
 qdbus $REF org.freedesktop.DBus.Properties.Set "" maximum $SIZE
```

Comment:
For `zenity` it needs passing progress on its stdin, not via dbus. This is already done in `qvm-copy-to-vm.gnome`. Hmm, maybe for this reason it would be better to fallback to that script, instead of adding `zenity` support here?